### PR TITLE
basic_text_classification colab example

### PR DESCRIPTION
**Pull request by sgwhat**
1. Add basic_text_classification
2. Add test script

### CODE REVIEW

File: docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "PnqB6ax-EPoB"
+      },
+      "source": [
+        "#\n",
+        "# Copyright 2018 Analytics Zoo Authors.\n",
```
Comment (hkvision) [inline docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb:38]: Do we need all these license? @Le-Zheng

File: docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+        "    init_orca_context(cores=1, memory=\"2g\") # run in local mode\n",
+        "elif cluster_mode == \"k8s\":\n",
+        "    init_orca_context(cluster_mode=\"k8s\", num_nodes=2, cores=4) # run on K8s cluster\n",
+        "elif cluster_mode == \"yarn\":\n",
+        "    init_orca_context(\n",
+        "        cluster_mode=\"yarn-client\", cores=4, num_nodes=2, memory=\"2g\",\n",
+        "        driver_memory=\"10g\", driver_cores=1,\n",
+        "        conf={\"spark.rpc.message.maxSize\": \"1024\",\n",
+        "              \"spark.task.maxFailures\": \"1\",\n",
+        "              \"spark.driver.extraJavaOptions\": \"-Dbigdl.failure.retryTimes=1\"}) # run on Hadoop YARN cluster"
```
Comment (hkvision) [inline docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb:288]: remove these conf

File: docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5mo8uojl9rQN"
+      },
+      "source": [
+        "## **Distributed Kreas using Orca APIs**\n",
```
Comment (hkvision) [inline docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb:237]: typo for Keras

File: docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yyRtP6lqieej"
+      },
+      "source": [
+        "Now, the accuracy of this model has reached 85%."
```
Comment (hkvision) [inline docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb:535]: around 85%

File: docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb (hunk 1 above)
Comment (Le-Zheng) [reply docs/docs/colab-notebook/orca/examples/basic_text_classification.ipynb:38]: I think we need it.

**Comment by sgwhat** [comment]
jenkins passed http://10.239.176.111:18888/job/ZOO-PR-Quickstart-notebook/

**Pull request merged**